common/test/acceptance/pages/lms/video/video.py

```diff
@@ -595,16 +598,12 @@ def downloaded_transcript_contains_text(self, transcript_format, text_to_search)
             'txt': 'text/plain',
         }
 
-        transcript_url_selector = self.get_element_selector(VIDEO_BUTTONS['download_transcript'])
-        url = self.q(css=transcript_url_selector).attrs('href')[0]
+        url = self.q(css=transcript_selector).attrs('href')[0]
         result, headers, content = self._get_transcript(url)
 
         if result is False:
             return False
 
-        if formats[transcript_format] not in headers.get('content-type', ''):
```

Comment:
I am concerned about losing this test coverage. Please figure out why the test is flaky (it may indicate an actual problem that a user would see).